src/robot/utils/robottime.py

```diff
@@ -39,10 +39,13 @@ def _float_secs_to_secs_and_millis(secs):
     return (isecs, millis) if millis < 1000 else (isecs+1, 0)
 
 
-def timestr_to_secs(timestr, round_to=3):
+def timestr_to_secs(timestr, round_to=3, accept_plain_values=True):
     """Parses time like '1h 10s', '01:00:10' or '42' and returns seconds."""
     if is_string(timestr) or is_number(timestr):
-        for converter in _number_to_secs, _timer_to_secs, _time_string_to_secs:
+        converters = [_timer_to_secs, _time_string_to_secs]
+        if accept_plain_values:
+            converters.insert(0, _number_to_secs)
```

Comment:
This util may be called pretty often and `insert` is slow. This would be faster and also simpler: 
```python
if accept_plain_values:
    converters = [_number_to_secs, _timer_to_secs, _time_string_to_secs]
else:
    converters = [_timer_to_secs, _time_string_to_secs]
```
Should also add unit test for this new functionality.